changelog.d/5863.bugfix

```diff
@@ -0,0 +1,3 @@
+Fix Synapse looking for config options `password_reset_failure_template` and
+`password_reset_success_template`, when they are actually
```

Comment:
this probably wants to be on one line, otherwise github mangles the final release notes